Classroom_mimic.nlogo

```diff
@@ -100,7 +140,10 @@ to read-patches-from-csv
 
     if reduce and (map is-number? row) [ ; only parse rows that contain only numbers
       set current_class item 2 row
-      if prev_class = -1 [set prev_class current_class]
+      if prev_class = -1 [
+        set Class_list lput current_class Class_list
+        set prev_class current_class
+      ]
```

Comment:
These changes fix #6 which became apparent when specifying classes in BehaviourSpace.